src/figuresClasses.ts

```diff
@@ -1,19 +1,82 @@
+enum Shapes {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+type Colors = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shapes,
+  color: Colors,
+  getArea(): number,
+}
 
+function validSides(...sides: number[]): void {
```

Comment:
```suggestion
function validateSides(...sides: number[]): void {
```
Describe action, not value